src/In.ProjectEKA.HipService/Common/HttpRequestHelper.cs

```diff
@@ -29,12 +31,19 @@ public static class HttpRequestHelper
             {
                 Content = new StringContent(json, Encoding.UTF8, MediaTypeNames.Application.Json)
             };
-            httpRequestMessage.Headers.Add(HeaderNames.Authorization, token);
+            if (token != null)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6377136)